athf/mcp/server.py

```
@@ -0,0 +1,79 @@
+
+
+def create_server(workspace_path: Optional[str] = None) -> FastMCP:
+    """Create and configure the ATHF MCP server.
+
+    Args:
+        workspace_path: Explicit workspace path (optional).
+
+    Returns:
+        Configured FastMCP server instance.
+    """
+    global _workspace
```

Ruff (0.15.6)

[warning] Using the global statement to update `_workspace` is discouraged

(PLW0603)